src/koclient/ko-view-model.ts

```diff
@@ -1,5 +1,6 @@
-import Utils = require("./utils");
-import Model = require("./model");
+import * as utils from "../common/utils";

```

Comment:
Heeeeeeeeeeeeeeeeeeeeeere is what I was looking for! I could make it run and find this. Thank you.